dapp/src/components/page/proposal/VotingResult.tsx

```diff
@@ -11,17 +11,16 @@ interface Props {
 const VotingResult: FC<Props> = ({ voteStatus, withDetail }) => {
   const voteResult = useMemo(() => {
     if (voteStatus) {
-      const sortedVoteResult = [
-        VoteType.APPROVE,
-        VoteType.CANCEL,
-        VoteType.REJECT,
-      ]
-        .map((voteType) => ({
-          type: voteType,
-          score: voteStatus[voteType].score,
-        }))
-        .sort((a, b) => b.score - a.score);
-      return sortedVoteResult[0]?.type;
+      let voteResult: VoteType | undefined = undefined;
+      const { approve, abstain } = voteStatus;
+      if (approve.score > abstain.score) {
+        voteResult = VoteType.APPROVE;
+      } else if (approve.score < abstain.score) {
+        voteResult = VoteType.REJECT;
+      } else {
+        voteResult = VoteType.CANCEL;
+      }
```

Comment:
This is wrong. I remember sharing this with you when you implemented the current live version. We have that in Rust.

```rust
if voted_approve > (voted_abstain + voted_reject) {
    proposal.status = types::ProposalStatus::Approved;
} else if voted_reject > (voted_abstain + voted_approve) {
    proposal.status = types::ProposalStatus::Rejected;
} else {
    proposal.status = types::ProposalStatus::Cancelled
}
```

`const { approve, abstain } = voteStatus;` here we must also look at the rejected votes.